bc.py

```diff
@@ -97,6 +97,18 @@ def add_pa(s, orders):
     r = s.get('https://www.bike-components.de/en/checkout/finalize/')
     doc = lxml.html.document_fromstring(r.text)
 
+    if not doc.cssselect('#voucher__token'):
```

Comment:
Ik denk dat je beter kan checken of de url `https://www.bike-components.de/en/checkout/payment/` is of dat er `Please select a payment method` op de pagina staat. Dat maakt het wat explicieter dan kijken of voucher token niet bestaat.

Reply:
Ik kan dit niet reproduceren omdat ik al een voorkeursbetalingsmethode heb geselecteerd, of ik moet een nieuw account aanmaken.